cmd/system-probe/config/config.go

```diff
@@ -153,6 +154,9 @@ func load() (*Config, error) {
 	if cfg.GetBool(nskey("ebpf_check", "enabled")) {
 		c.EnabledModules[EBPFModule] = struct{}{}
 	}
+	if cfg.GetBool("language_detection.enabled") {
+		c.EnabledModules[LanguageDetectionModule] = struct{}{}
+	}
```

Comment:
can be done later, but I suggest adding a section in `pkg/config/config_template.yaml`.
that's super useful for non containerized environments

Reply:
This file seems like it's for the `datadog.yaml`, no? The `languaeg_detection.enabled` setting is in the `system-probe.yaml`. Surprisingly, I don't see a similar file for `system-probe.yaml`

Reply:
Ah I see. In that case I will update the template